pkg/kubelet/kubelet.go

```diff
@@ -696,14 +696,14 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 		var ips []net.IP
 		cfgAddress := net.ParseIP(kubeCfg.Address)
 		if cfgAddress == nil || cfgAddress.IsUnspecified() {
-			if localIPs, err := allLocalIPsWithoutLoopback(); err != nil {
+			localIPs, err := allLocalIPsWithoutLoopback()
+			if err != nil {
 				return nil, err
-			} else {
-				ips = localIPs
 			}
-		} else {
-			ips = []net.IP{cfgAddress}
+			ips = localIPs
 		}
+		ips = []net.IP{cfgAddress}
```

Comment:
This line should only be run if `cfgAddress == nil || cfgAddress.IsUnspecified()` is not true. Now it always runs regardless.